client/my-sites/activity/activity-log/rewind-unavailability-notice.jsx

```diff
@@ -33,7 +33,7 @@ export const RewindUnavailabilityNotice = ( {
 					href={ adminUrl }
 					title={ translate( 'The site is not connected.' ) }
 					description={ translate(
-						"We can't back up or rewind your site until it has been reconnected."
+						"We can't back up or restore your site until it has been reconnected."
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.